VK_App/VK_App/ViewControllers/FirstBarVC/FriendsTableViewController.swift

```diff
@@ -33,19 +40,33 @@ final class FriendsTableViewController: UITableViewController {
 
     // MARK: Private properties
 
-    private var sectionsDict: [Character: [(String, String)]] = [:]
+    private var sectionsDict: [Character: [(String, String, [String]?)]] = [:]
```

Comment:
```suggestion
    private var sectionsDict: [Character: [(String, String, [String]?)]] = [:]
```
Можно typealias сделать, но это по желанию